### PR TITLE
Add e2e tests for disable plugin command

### PR DESCRIPTION
**Pull request by akshaychhajed**
#### Summary
Add e2e tests for disable plugin command

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/16078

### CODE REVIEW

File: commands/mmctl_test.go [hunk 1]
```diff
@@ -91,3 +91,11 @@ func (s *MmctlE2ETestSuite) RunForAllClients(testName string, fn func(client.Cli
 		fn(s.th.LocalClient)
 	})
 }
+
+// RunUnprivilegededClient runs a test using unauthenticated client
```
Comment (agnivade) [inline commands/mmctl_test.go:95]: typo

**Review by agnivade — COMMENTED** [review]
Nice work @akshaychhajed ! Just a small thing to fix.

File: commands/mmctl_test.go (hunk 1 above)
Comment (akshaychhajed) [reply commands/mmctl_test.go:95]: Oops. Let me fix it and update PR.

**Review by mgdelacroix — APPROVED** [review]
LGTM, great work @akshaychhajed! Thanks for the contribution!

File: commands/mmctl_test.go (hunk 1 above)
Comment (agnivade) [reply commands/mmctl_test.go:95]: Hey @akshaychhajed - how is it going with this one? :)

Comment (akshaychhajed) [reply commands/mmctl_test.go:95]: I received comments for my enable plugin PR which may be relevant to this pull request too.

https://github.com/mattermost/mmctl/pull/274#discussion_r518806300

I wanted to test it with unprivileged user not in local mode.
I will need to review the different modes for the client to get a better picture.

**Comment by agnivade** [comment]
Please feel free to request my review when it's ready.

**Comment by akshaychhajed** [comment]
@agnivade ok. I will.

**Comment by jasonblais** [comment]
@akshaychhajed Let us know if you have any questions about the previous feedback, we'd be happy to help :)

**Comment by akshaychhajed** [comment]
@jasonblais i am away from keyboard and be back over weekend. I will update when i am back.

**Comment by jfrerich** [comment]
Hi, @akshaychhajed - checking in on this PR also to see if you've made progress or need any additional help  :)

**Comment by jfrerich** [comment]
Hi @akshaychhajed! Just checking in again to see if you need any help. I'd love to get this merged, or open it back up for another contributor

**Comment by akshaychhajed** [comment]
@jfrerich I want to get this merged too. Sorry, I was away from keyboard.

**Comment by jfrerich** [comment]
No worries, @akshaychhajed! I was just checking in to make sure the PR wasn't abandoned :) Let us know if you need any assistance!

**Comment by emilyacook** [comment]
Hey @akshaychhajed! Let me know if you have time to take action on this, otherwise I'll need to update the status as per our [inactive contribution guidelines](https://developers.mattermost.com/contribute/getting-started/inactive-contributions/)

**Comment by akshaychhajed** [comment]
Thank you for your patience!
I updated PR with requested changes.

I could not get the pluginID for test. I have marked the plugin constant below
https://github.com/mattermost/mmctl/pull/275/files#diff-cd6aeaed48e9efc5c7a1b72d620073ca953d5e724099f7bc74206a33a2a81afcR18

**Comment by mgdelacroix** [comment]
@akshaychhajed can you please take a look at the test that is failing in the CI?

```
    --- FAIL: TestMmctlE2ESuite/TestPluginDisableCmd (5.04s)
    plugin_e2e_test.go:156: 
        	Error Trace:	plugin_e2e_test.go:156
        	            				plugin_e2e_test.go:27
        	Error:      	Expected nil, but got: invalid url /go/src/github.com/mattermost/mattermost-server/tests/testplugin.tar.gz
        	Test:       	TestMmctlE2ESuite/TestPluginDisableCmd
```

**Comment by akshaychhajed** [comment]
@mgdelacroix sure. Let me take a look.

**Comment by akshaychhajed** [comment]
@mgdelacroix I used the path @isacikgoz suggested [here](https://github.com/mattermost/mmctl/pull/274#discussion_r518807570)

Unfortunately DownloadFromUrl expects a http path.
https://github.com/mattermost/mattermost-server/blob/8d6dd195e1d8cdcd2ed7aaa12da95125cc2033c4/app/download.go#L26

Is there http path for test plugin? 
Also I need test plugin id.

**Review by agnivade — APPROVED** [review]

**Comment by hanzei** [comment]
Thank you for your contribution. :+1:  `mmctl` has been moved into https://github.com/mattermost/mattermost/tree/master/server/cmd/mmctl.  Please resubmit your PR to https://github.com/mattermost/mattermost.

**Pull request closed without merging**